src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -394,7 +394,9 @@ Creating a custom `public/_routes.json` will override the automatic generation.
 
 The Astro [Sessions API](/en/guides/sessions/) allows you to easily store user data between requests. This can be used for things like user data and preferences, shopping carts, and authentication credentials. Unlike cookie storage, there are no size limits on the data, and it can be restored on different devices. 
 
-Astro automatically configures [Workers KV](https://developers.cloudflare.com/kv/) for session storage when using the Cloudflare adapter. Before using sessions, you need to create a KV namespace to store the data and configure a KV binding in your Wrangler config file. By default, Astro expects the KV binding to be named `SESSION`, but you can choose a different name if you prefer by setting the [`sessionKVBindingName`](#sessionkvbindingname) option in the adapter config.
+Astro automatically configures [Workers KV](https://developers.cloudflare.com/kv/) for session storage when using the Cloudflare adapter in production. To avoid mistakes that could compromise your data or cause unexpected bills, the filesystem storage will be used during local development by default.
```

Comment:
It's not to avoid mistakes. It's because we don't have access to KV when running in dev, because `astro dev` doesn't use wrangler. The same applies to the Netlify adapter when running without the Netlify CLI

Reply:
@ascorbic You are right we don't have access to remote KV. But we do have access to Cloudflare's Proxy / Stubs. There shouldn't be any code we need to have for `dev`, production code accessing KV, should also work locally.

Reply:
Thanks @alexanderniebuhr. That would be great.

Reply:
@sarah11918, this PR probably won't be needed if we change Sessions to use the local `KV` that is available in the Cloudflare adapter. Since Alex mentioned he'd work through this issue during TBD I'd rather have this one sitting and possibly closed as soon as this resolves than to merge and then a few days later we change it again.